mslib/msui/sideview.py

```diff
@@ -39,9 +39,11 @@
 from mslib.utils import thermolib
 from mslib.utils.config import config_loader
 from mslib.utils.units import units, convert_to
+from mslib.msui import autoplot_gui as gui
```

Comment:
when it is now a dockwidget, this should have a similiar name as the others, e.g. autoplot_dockwidget.py

as auto_dock maybe